app/assets/javascripts/image.js

```diff
@@ -0,0 +1,47 @@
+function loadImages() {
+  var item = $('#image_uuid');
+  var url = item.data('url');
+  var selected = item.data('selected');
+
+  item.indicator_show();
+  foreman.tools.showSpinner();
+  item.attr("disabled", true);
+
+  $.ajax({
+    type: 'get',
+    url: url,
+    complete: function() {
+      foreman.tools.hideSpinner();
```

Comment:
I am also not sure if core will be willing to merge newly added js file in the 'old structure' under app/assets/javascripts since Webpack was introduced. You may consider moving it under webpack/assets/javascripts.